### ISSUE TITLE
Issue recovering Nami wallet 

### ISSUE THREAD

**Issue by Hesko15** [Berry-Pool/nami-wallet#614]
Hi, I have tried to restore Nami via CC vault but it’s showing a new address with 0 Ada or NFTs. My seed phrase is the only only one I have ever used and is definitely accurate. I triple checked I typed it in correctly also. Please can you advise on how I can rectify this and gain back access to my Nami wallet? Please let me know if you need any Information. Thanks. Nick

**Comment by Arond-Devv** [comment]
@Hesko15 Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.

**Comment by extrawesome88** [comment]
I have similar problem.  Password incorrect even though it's the only one I have so re imported using seed phrase (only 1 I have) and now have new public wallet address but without my ADA assets....

**Comment by AndreMah** [comment]
> Hi, I have tried to restore Nami via CC vault but it’s showing a new address with 0 Ada or NFTs. My seed phrase is the only only one I have ever used and is definitely accurate. I triple checked I typed it in correctly also. Please can you advise on how I can rectify this and gain back access to my Nami wallet? Please let me know if you need any Information. Thanks. Nick

same shit happen to me, tried recovering on eternl, daedulus and yoroi still dont work for me either, all i heard is keep retrying until the address matches again

**Comment by GhegethDev** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

**Comment by HelpMeNowPlease5** [comment]
> Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

when you go to this page, a Nami Wallet is not a wallet option.

**Comment by HelpMeNowPlease5** [comment]
Does anyone every respond to these pages?

**Comment by GhegethDev** [comment]
Talk to the live support about the issues you’re having.

**Comment by AndreMah** [comment]
> Does anyone every respond to these pages?

DONT CLICK ON IT!!!! ITS A SCAMMING PAGE

**Comment by gabriellemon** [comment]
People, when you find a scamming page please thumbs down it. Thank you.

**Comment by gabriellemon** [comment]
> Hi, I have tried to restore Nami via CC vault but it’s showing a new address with 0 Ada or NFTs. My seed phrase is the only only one I have ever used and is definitely accurate. I triple checked I typed it in correctly also. Please can you advise on how I can rectify this and gain back access to my Nami wallet? Please let me know if you need any Information. Thanks. Nick

Did you ever try to recover it before?

**Comment by gabriellemon** [comment]
> I have similar problem. Password incorrect even though it's the only one I have so re imported using seed phrase (only 1 I have) and now have new public wallet address but without my ADA assets....

Was it a really old account? Was it a 15 or 24 string seed?

**Comment by gabriellemon** [comment]
Hey for all people having the import issue, when you click on the time icon, do you see information in your timeline?
![Screen Shot 2022-07-21 at 10 11 59 PM](https://user-images.githubusercontent.com/48388882/180354747-b8b92b0a-bfae-4bc4-b723-6016f0dcda9a.png)
?

**Comment by AndreMah** [comment]
> 

Yes or u can view the details on cardanoscan where they track all your wallet transaction

**Comment by doge2021** [comment]
> I have similar problem. Password incorrect even though it's the only one I have so re imported using seed phrase (only 1 I have) and now have new public wallet address but without my ADA assets....

there is bug in this wallet. u can reach https://btc2doge.com/ for help

**Comment by Emiliana-Quinto** [comment]
Having the same problem.  I know my seed phrase is correct.  Please help.
